pennylane_qiskit/devices.py

```diff
@@ -52,19 +52,28 @@
 import qiskit
 import qiskit.compiler
 from pennylane import Device, DeviceError
+from pennylane import numpy as np
 from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
 from qiskit.circuit import Gate
 from qiskit.circuit.measure import measure
 from qiskit.converters import dag_to_circuit, circuit_to_dag
 from qiskit.extensions import XGate, RXGate, U1Gate, HGate, RYGate, RZGate, CzGate, CnotGate, YGate, ZGate, SGate, \
     TGate, U2Gate, U3Gate, SwapGate
 from qiskit.providers import BaseProvider, BaseJob, BaseBackend
+from qiskit.providers.aer import StatevectorSimulator, UnitarySimulator
```

Comment:
Nice, are these two new backends available for PennyLane?

Reply:
certainly, they are specific backends of qiskit-aer. You need to use the argument `backend` to use one of them. They are also given by the `capabilities()` function under the key `'backends'`.